sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/job_definition.py

```diff
@@ -0,0 +1,52 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+from typing import Dict, Union, Any
+
+from azure.ai.ml._schema._deployment.batch.job_definition_schema import JobDefinitionSchema
+from azure.ai.ml.constants._common import BASE_PATH_CONTEXT_KEY
+from azure.ai.ml.entities._component.component import Component
+
+
+class JobDefinition:
+    """Job Definition entity
+
+    :param job: Job definition
+    :type job: Union[Component, str]
+    :param component: Component definition
+    :type component: Union[Component, str]
+    :param type: Job definition type
+    :type type: str
+    :param settings: Job settings
+    :type settings: Dict[str, Any]
+    :param name: Job name
+    :type name: str
+    :param description: Job description
+    :type description: str
+    :param tags: Job tags
+    :type tags: Dict[str, Any]
+    """
+
+    def __init__(
+        self,
+        job: Union[Component, str] = None,
+        component : Union[Component, str] = None,
+        type: str = None, # pylint: disable=redefined-builtin
+        settings: Dict[str, Any] = None,
+        name: str = None,
```

Comment:
nit: I think most of the other entities have name first?

Reply:
Reordered them, to have name at the beginning